src/panel/applets/tray/DBusMenuNode.vala

```diff
@@ -378,4 +378,13 @@ private class Properties {
 
 		return ret;
 	}
+
+	private static Variant? unbox(Variant? variant) {
+		int levels = 0; // insurance policy
```

Comment:
I'm curious, why 8?

Reply:
Oh, wrong line. Oh well, you know what I mean.

Reply:
Didn't want it to be infinite, but also didn't want to only unbox once, just in case. 8 seemed like a decent cap.